src/components/Message/MessageSimple/MessageContent.js

```diff
@@ -93,683 +83,576 @@ const FailedText = styled.Text`
+  /**
+   * Custom UI component to display image attachments.
+   * Defaults to https://github.com/GetStream/stream-chat-react-native/blob/master/src/components/Gallery.js
+   */
+  Gallery: PropTypes.oneOfType([PropTypes.node, PropTypes.elementType]),
+  /**
+   * Custom UI component to display Giphy image.
+   * Defaults to https://github.com/GetStream/stream-chat-react-native/blob/master/src/components/Card.js
+   */
+  Giphy: PropTypes.oneOfType([PropTypes.node, PropTypes.elementType]),
+  /**
+   * Position of message in group - top, bottom, middle, single.
+   *
+   * Message group is a group of consecutive messages from same user. groupStyles can be used to style message as per their position in message group
+   * e.g., user avatar (to which message belongs to) is only showed for last (bottom) message in group.
+   */
+  groupStyles: PropTypes.array,
+  /** Handler for actions. Actions in combination with attachments can be used to build [commands](https://getstream.io/chat/docs/#channel_commands). */
+  handleAction: PropTypes.func,
+  /**
+   * Handler to delete a current message.
+   */
+  handleDelete: PropTypes.func,
+  /**
+   * Handler to edit a current message. This message simply sets current message as value of `editing` property of channel context.
+   * `editing` prop is then used by MessageInput component to switch to edit mode.
+   */
+  handleEdit: PropTypes.func,
+  // enable hiding reaction count from reaction picker
+  hideReactionCount: PropTypes.bool,
+  // enable hiding reaction owners from reaction picker
+  hideReactionOwners: PropTypes.bool,
+  /** Object specifying rules defined within simple-markdown https://github.com/Khan/simple-markdown#adding-a-simple-extension */
+  markdownRules: PropTypes.object,
+  /**
+   * Array of allowed actions on message. e.g. ['edit', 'delete', 'reactions', 'reply']
+   * If all the actions need to be disabled, empty array or false should be provided as value of prop.
+   * */
+  messageActions: PropTypes.oneOfType([PropTypes.bool, PropTypes.array]),
+  MessageFooter: PropTypes.oneOfType([PropTypes.node, PropTypes.elementType]),
+  MessageHeader: PropTypes.oneOfType([PropTypes.node, PropTypes.elementType]),
+  MessageReplies: PropTypes.oneOfType([PropTypes.node, PropTypes.elementType]),
+  /** Custom UI component for message text */
+  MessageText: PropTypes.oneOfType([PropTypes.node, PropTypes.elementType]),
+  /**
+   * Function that overrides default behaviour when message is long pressed
+   * e.g.
+   *
+   * if you would like to open reaction picker on message long press:
+   *
+   * ```
+   * import { MessageSimple } from 'stream-chat-react-native' // or 'stream-chat-expo'
+   * ...
+   * const MessageUIComponent = (props) => {
+   *  return (
+   *    <MessageSimple
+   *      {...props}
+   *      onLongPress={(message, e) => {
+   *        props.openReactionPicker();
+   *        // Or if you want to open action sheet
+   *        // showActionSheet();
```

Comment:
where is `showActionSheet` coming from? Its not in props of MessageSimple

Reply:
i see what you're saying, showActionSheet is defined within MessageContent, i'll move it up so it can be pulled off props

Reply:
@vishalnarkhede ok all updated